pom.xml

```diff
@@ -195,6 +195,7 @@
                 </executions>
                 <configuration>
                     <includes>
+                        <!-- TODO what to do with this part? -->
```

Comment:
good question, need to check. I guess we just copied that from somewhere for maven central release.